src/jit/emit.cpp

```diff
@@ -783,9 +783,10 @@ insGroup* emitter::emitSavIG(bool emitAdd)
 
     // printf("Group [%08X]%3u has %2u instructions (%4u bytes at %08X)\n", ig, ig->igNum, emitCurIGinsCnt, sz, id);
 
-    /* Record the live GC register set - if and only if it is not an emitter added block */
+    /* Record the live GC register set - if and only if it is not an extension block,
```

Comment:
Minor: When we "touch" these C-style comments we like to update them to `//` comments

Reply:
Done, I have updated all comments in this function to // style.